packages/rich-text/src/plugins/Hr/index.tsx

```diff
@@ -101,6 +102,8 @@ export function ToolbarHrButton(props: ToolbarHrButtonProps) {
     Slate.ReactEditor.focus(editor);
   }
 
+  if (!editor) return null;
```

Comment:
What are these changes for?

Reply:
Lint was accusing some errors when I was trying to improve the internal type. `editor` might be undefined. We also used the same pattern on other buttons but forgot to add on some. I can remove it if that's causing noise and confusion.

Reply:
👍

Reply:
(👍 to keeping it, not removing)